api/src/main/java/com/example/api/service/activity/task/TaskService.java

```diff
@@ -120,4 +142,177 @@ public List<ActivitiesResponse> getAllActivities() {
                 .flatMap(Collection::stream)
                 .toList();
     }
+
+    public List<RequirementResponse<?>> getRequirementForActivity(Long id) throws EntityNotFoundException, MissingAttributeException {
+        Activity activity = getActivity(id);
+        List<Requirement> requirements = activity.getRequirements();
+        requirements = requirements.stream()
+                .sorted(Comparator.comparingInt(o -> o.getType().getType()))
+                .toList();
```

Comment:
Sortowanie jest tu konieczne?

Reply:
nie jest, ale raz mi nie wiem czemu coś zle wyszlo i dodalem to